src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/Sms/sendChallenge.html.twig

```diff
@@ -25,17 +25,29 @@
     {% endif %}
 
     {{ form_start(form)|replace({'form-horizontal': ''})|raw }}
+    {{ form_errors(form) }}
     <div class="row">
-        {{ form_errors(form) }}
-        <div class="col-xs-8">
+        <div class="col-md-8">
+            {% if form.country.vars.errors|length %}
+                <div class="alert alert-danger">{{ form_errors(form.country) }}</div>
```

Comment:
Could you go into what prompted you to change the form error display? Wouldn't the errors for `country` be displayed twice now?